lib/core/crypto/eosdart/src/client.dart

```diff
@@ -216,25 +217,35 @@ class EOSClient extends NetworkingManager {
   }
 
```

Comment:
Here I split pushTransaction method in 2, one with and one without 'broadcast'

We currently never use broadcast = false, but who knows. Leaving it in for completeness. We can use this to do what Anchor is doing, authorizing by sending a signed transaction to our server, rather than making a full chain transaction.

Reply:
is this how they dismiss the QR code from the DAO view when the transaction is signed?